frontend/controller/actions/group.js

```diff
@@ -41,9 +42,20 @@ import { CONTRACT_HAS_RECEIVED_KEYS, EVENT_HANDLED } from '~/shared/domains/chel
 import type { Key } from '../../../shared/domains/chelonia/crypto.js'
 import { CURVE25519XSALSA20POLY1305, EDWARDS25519SHA512BATCH, keyId, keygen, serializeKey } from '../../../shared/domains/chelonia/crypto.js'
 import type { GIActionParams } from './types.js'
+import { REMOVE_NOTIFICATION } from '~/frontend/model/notifications/mutationKeys.js'
 import { createInvite, encryptedAction } from './utils.js'
 import { extractProposalData } from '@model/notifications/utils.js'
 
+sbp('okTurtles.events/on', LEFT_GROUP, ({ identityContractID, groupContractID }) => {
+  const rootState = sbp('state/vuex/state')
+  if (rootState.loggedIn?.identityContractID !== identityContractID) return
+  // NOTE: remove all notifications whose scope is in this group
+  // TODO: FIND ANOTHER WAY OF DOING THIS WITHOUT ROOTGETTERS
```

Comment:
Does this comment still make sense now?

Reply:
Yes, until I figure it out. Like mentioned, the main issue is that root anything shouldn't be used in contracts (for sandboxing among other things). However, actions (which will run in the SW) ideally shouldn't need Vuex to be defined. For documentation purposes of what's been discussed, the options seem to be:

1. Refactoring so that getters are not used
2. A Vuex-lite module in the SW that defines used getters
3. Importing Vuex and all getters

Reply:
I like (2).... there can be a shared filed that's imported in both the browser and sw that uses these getters.